### PR TITLE
remove vampire from charged black extract

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
dont want to give people a race that forces them to suck blood and potentially kill people to stay alive
#### Changelog

:cl:  
rscdel: Removed vampire from charged black 
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
ill fix trhis if i remeber tag me or something

File: code/modules/mob/living/carbon/human/species_types/vampire.dm [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	inherent_traits = list(TRAIT_NOHUNGER,TRAIT_NOBREATH)
 	inherent_biotypes = list(MOB_UNDEAD, MOB_HUMANOID)
 	default_features = list("mcolor" = "FFF", "tail_human" = "None", "ears" = "None", "wings" = "None")
-	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | ERT_SPAWN | SLIME_EXTRACT
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | ERT_SPAWN |
```
Comment (Hopekz) [inline code/modules/mob/living/carbon/human/species_types/vampire.dm]:
```suggestion
	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | ERT_SPAWN
```
To make it compile

**Review by Hopekz — CHANGES_REQUESTED** [review]

**Comment by overlord12345** [comment]
The vamp race doesn't need to suck human blood, they can suck on monkeys plus blood recharges after you use bat form.

**Comment by boodaliboo** [comment]
> The vamp race doesn't need to suck human blood, they can suck on monkeys plus blood recharges after you use bat form.

still shouldnt give people something that makes you assume your a antag

**Pull request merged**